src/pages/index.page.jsx

```diff
@@ -11,10 +11,11 @@ export default function Home() {
             <div className="home-top__text-column">
               <h1>Accessibility Cheatsheets for Components</h1>
               <p className="home-top__top-description">
-                Intro copy goes here, such as: Build accessible components more
-                accurately and in less time with these easy-to-read cheatsheets
-                based on WCAG and ARIA best practices. Check back often as
-                we&#39;re adding new components frequently!
+                Building accessible components is hard: the requirements are
+                complicated, the specifications are dense, and good examples are
+                sparse. These cheatsheets should make that a little bit
+                easier—and they&#39;re based on WCAG and ARIA best practices.
+                Check back often as we&#39;re adding new components frequently!
               </p>
```

Comment:
Let's split this into two paragraphs. The content should be good, though!

```suggestion
<p className="home-top__top-description">
  Building accessible components is hard: the requirements are
  complicated, the specifications are dense, and good examples are
  sparse.
</p>
<p className="home-top__top-description">
  These cheatsheets should make that a little bit
  easier—and they&#39;re based on WCAG and ARIA best practices.
  Check back often as we&#39;re adding new components frequently!
</p>
```

Reply:
Thanks Dustin, I did the paragraph break!

Reply:
Is `home-top__top-description` doing anything? I just looked at the deploy preview and that class doesn't appear to apply any styles. We can probably remove the `className` if that's the case.

Reply:
I checked, and it's doing nada--so it's gone now. Thanks for spotting it!